README.md

```diff
@@ -33,7 +33,11 @@ node app
 
 ## Run
 
-Refresh the DataLake views. The views give fast access to enrollment and assignment data.
+Refresh the DataLake views. The materialized views are intended to give fast access to analytics data to the application layer.
```

Comment:
@sandeepmjay The sentence above is not clear. How about, "These high-performance materialized views give applications access to analytics data."

Reply:
sounds good. Flipping it.